src/diff.jl

```diff
@@ -791,12 +790,13 @@ end
 #######################################################################################################################
 # Vector Calculus
 #######################################################################################################################
-abstract type ArrayOperator end
 
-struct ArrayDifferentialOperator <: ArrayOperator
```

Comment:
I think you can make this a subtype of `Operator`

Reply:
done